cdap-ui/app/cdap/services/helpers.js

```diff
@@ -337,6 +338,17 @@ const convertKeyValuePairsToMap = (keyValuePairs, ignoreNonNilValues = false) =>
   return map;
 };
 
+const roundDecimalToNDigits = (num, digits = 2) => {
```

Comment:
shorten name to be `roundToNDigits`?

Reply:
So I explicitly had the decimal part to the name to indicate the rounding happens after the decimal and not the entire number. Underneath it uses `Number.toFixed` instead of `Math.round`, which is used only for decimal numbers.